packages/syntax-core/src/Typography/Typography.stories.tsx

```diff
@@ -21,28 +21,43 @@ export default {
       control: { type: "radio" },
     },
     color: {
+      // | "gray900"
+      // | "gray700"
+      // | "primary"
+      // | "destructive-primary"
+      // | "destructive-darkBackground"
+      // | "success"
+      // | "success-darkBackground"
+      // | "white"
+      // | "inherit",
```

Comment:
is this comment block intentional? If so, is it meant to maintain documentation on classic colors?

Reply:
@zprong removed - thank you